examples/v1beta1/trial-images/pytorch-mnist/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM pytorch/pytorch:1.0-cuda10.0-cudnn7-runtime
+# TODO (tenzen-y): Upgrade Python version and Pytorch version
+FROM python:3.7-slim
```

Comment:
Why is the version different from the others?

Reply:
We need to fix the pytorch-mnist example to use Python v3.9 since both torch and torchvision can not execute in Python 3.9 as described. in [this](https://pypi.org/project/torchvision/).

> torch | torchvision | python
> -- | -- | --
> <=1.0.1 | 0.2.2 | ==2.7, >=3.5, <=3.7



So I will create another PR to upgrade the Python version after this PR merging.

Reply:
FYI: https://github.com/kubeflow/katib/issues/1883